libmscore/input.cpp

```diff
@@ -116,7 +116,9 @@ void InputState::moveInputPos(Element* e)
             return;
 
       Segment* s;
-      if (e->isChordRest())
+      if (e->isSegment())
```

Comment:
Updated to follow new guide lines and removed the else branch.
I added a Q_ASSERT to make sure s has been assigned a valid pointer.